Plugins/Effect/Effect.cpp

```diff
@@ -285,4 +306,142 @@ ArgumentStack Effect::ReplaceEffect(ArgumentStack&& args)
     return found;
 }
 
+ArgumentStack Effect::GetTrueEffectCount(ArgumentStack&& args)
+{
+    int32_t retVal = 0;
+    auto objectId = Services::Events::ExtractArgument<ObjectID>(args);
+
+    if(objectId != Constants::OBJECT_INVALID)
```

Comment:
Nit: no need for this check, `GetGameObject(OBJECT_INVALID)` will return nullptr, and AsNWSObject() of that will also be null. Same in other instances of this pattern.